api/models/age_ranges.py

```diff
@@ -3,21 +3,28 @@
 from api.models.constants import DEFAULT_CHOICE_CHAR_FIELD_MAX_LENGTH
 
 
+# defining outside of model to use in data migration
```

Comment:
I think this comment also is not necessary, because you can always check in code where class is used :)

Reply:
OK